chart/values.yaml

```diff
@@ -100,4 +101,4 @@ nodeSelector: {}
 
 tolerations: []
 
-affinity: {}
+affinity: {}
```

Comment:
Please re-add the newline at the end (and check your IDE configuration)

Reply:
It seems like I accidentally removed it. Just added it again in the new commit.